rarbgapi/rarbgapi.py

```diff
@@ -127,6 +127,8 @@ def wrapper(self, *args, **kwargs):
                 resp = func(self, token=self._token, *args, **kwargs)
                 json_ = resp.json(object_hook=json_hook)
                 return json_['torrent_results']
+            except KeyError:
```

Comment:
Yes, but still I would like to receive an empty list if nothing was found. Do you not like that?